packages/tools/src/tools/annotation/EllipticalROITool.ts

```diff
@@ -1112,9 +1112,21 @@ class EllipticalROITool extends AnnotationTool {
           modalityUnitOptions
         );
 
+        const invXRadiusSq =
```

Comment:
Please remove this bit of code, and instead, just pass an empty object down below - let the internals of pointInEllipse assign the right value

Reply:
I don't want those to get recalculated, the whole point is to not recalculate on every check